src/client/Client.cc

```diff
@@ -12973,7 +12974,21 @@ int Client::ll_statfs(Inode *in, struct statvfs *stbuf, const UserPerm& perms)
   /* Since the only thing this does is wrap a call to statfs, and
      statfs takes a lock, it doesn't seem we have a need to split it
      out. */
```

Comment:
And those look great, so nice addition of statfs method with path.

Although Ganesha won't use it, you might want to add a path based ll_statfs_path and a ceph_ll_statfs_path.

Reply:
I see what you mean.

I'm more of a minimalist for adding/changing apis. I'm not aware of a hard requirement for those since the other statfs calls were added in 2013. Perhaps if it's required we add them later?